tests/integration/targets/elb_target/tasks/etg_protocol_version.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Run elb_target_group protocol_version tests
+  block:
+
+    # =====================================================================
+    - name: set up testing VPC
+      ec2_vpc_net:
+        name: "{{ resource_prefix }}-etg-protocol-version-vpc"
+        state: present
+        cidr_block: 20.0.0.0/16
+        tags:
+          Name: "{{ resource_prefix }}-etg-protocol-version-vpc"
+          Description: "Created by ansible-test"
+      register: vpc
+
+    # =====================================================================
+    - name: Create a target group with protocol_version 'GRPC'
+      community.aws.elb_target_group:
+        name: "{{ elb_target_group_name }}"
+        protocol: http
+        port: 80
+        vpc_id: "{{ vpc.vpc.id }}"
+        protocol_version: GRPC
+        state: present
+      register: create_result
+
+    - assert:
+        that:
+          - create_result is changed
+          - create_result is not failed
+          - create_result.protocol_version == "GRPC"
+          - create_result.protocol == "HTTP"
+          - create_result.port == 80
+
+    - name: Create a target group with protocol_version 'GRPC' (idempotence)
+      community.aws.elb_target_group:
+        name: "{{ elb_target_group_name }}"
+        protocol: http
+        port: 80
+        vpc_id: "{{ vpc.vpc.id }}"
+        protocol_version: GRPC
+        state: present
+      register: create_result
+
+    - assert:
+        that:
+          - create_result is not changed
+          - create_result is not failed
+          - create_result.protocol_version == "GRPC"
+          - create_result.protocol == "HTTP"
+          - create_result.port == 80
```

Comment:
@sharvarikhedkar can you also please a third test that verify that a change of the `protocol_version` works properly?

Reply:
...otherwise LGTM

Reply:
Hi @markuman, I tried testing the code for [updating the target group with different protocol_versions](https://github.com/ansible-collections/community.aws/pull/1496/commits/54b2128980451f7f7d07a84ec303d832a0e67f58) but [looking at the boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elbv2.html#ElasticLoadBalancingv2.Client.modify_target_group) and trying to modify `protocol_version` through AWS console, it seems like the update operation for `protocol_version` parameter is not supported yet, any insights on this?

Reply:
Oh, I didn't know that. Maybe that should be mentioned than in the module documentation @sharvarikhedkar

Reply:
Added.